tests/view_tests/tests/test_debug.py

```diff
@@ -375,6 +375,21 @@ def test_reporting_of_nested_exceptions(self):
         self.assertIn(implicit_exc.format("Second exception"), text)
         self.assertEqual(3, text.count('<p>Final exception</p>'))
 
+    def test_reporting_frames_without_source(self):
+        try:
+            source = "def funcName():\n    raise Error('Whoops')\nfuncName()"
```

Comment:
Use single quotes (unless a string contains a single quote) as described in [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).